src/components/change-requests/change-requests-table/change-requests-filter/change-requests-filter.module.css

```diff
@@ -26,12 +26,21 @@
   transform: translateY(-50%);
 }
 
-.applyButton {
-  float: right;
+.applyButton,
+.clearButton {
   background-color: #ef4345 !important;
   border-color: #ef4345 !important;
 }
 
-.applyButton:focus {
+.applyButton {
+  float: right;
+}
+
+.clearButton {
+  float: left;
+}
```

Comment:
You can eliminate the use of floating elements by utilizing flexbox. You'll need to create a container to put both the buttons in. When you create the container element, add a class onto it that has `display: flex`, `align-items: center`, and `justify-content: space-between` (I think these are all the right properties but I might've messed up some dashes or capitalization) and that will do the same thing as your floats but without taking those elements out of the normal HTML flow. You may need to set the width of the container to `100%` or something like that to get the spacing of the buttons correct.